graylog2-bootstrap/src/main/resources/log4j.xml

```diff
@@ -10,6 +10,10 @@
         </layout>
     </appender>
 
+    <appender name="memory" class="org.graylog2.MemoryAppender">
```

Comment:
The name should be more obviously point to the internal usage of this appender, e. g. "graylog-internal-logs", and have a short comment that this shouldn't be removed or changed.